src/components/RequestCards.tsx

```diff
@@ -13,17 +11,15 @@ export const RequestCards = (props: IProps) => {
 
   let card = showData.map((request) => {
     return (
-      <div>
-        <RequestCard
-          id={request.id}
-          title={request.title}
-          category={request.category}
-          upvotes={request.upvotes}
-          status={request.status}
-          description={request.description}
-          comments={request.comments}
-        />
-      </div>
+      <RequestCard
+        id={request.id}
```

Comment:
Not required, but one thing you can do to cut down some noise here is to de-structure. What I've written is the exact same thing:

```
  let card = showData.map((request) => {
   const {id, title, category, upvotes, status, description, comments } = request
    return (
      <div>
        <RequestCard
          id={id}
          title={title}
          category={category}
          upvotes={upvotes}
          status={status}
          description={description}
          comments={comments}
        />
      </div>
)}
```

1. Does this _need_ a wrapping div?
2. No need to set this up as a variable -- it's not being reused. It can go right inline in the return